source/ranges.tex

```diff
@@ -2268,10 +2268,10 @@
 \end{codeblock}
 
 \pnum
-Let \exposid{container-appendable} be defined as follows:
+Let \exposid{container-append} be defined as follows:
 \begin{codeblock}
 template<class Ref, class Container>
```

Comment:
```suggestion
template<class Container>
```
https://github.com/cplusplus/draft/commit/1958e5063b6c5c6893ba509eff02bf933e1a5e02 says:

> [range.utility.conv.general]p5 Template parameters are reversed from what the paper has; resolution applied to body but not to remove Ref.

Nonetheless, I think the only reasonable interpretation of [LWG 4016](https://cplusplus.github.io/LWG/issue4016) is to remove the template parameter `Ref`.

cc @jwakely

Reply:
Yes, obviously it needs to be removed. It is no longer used in the function body, and an explicit template argument is no longer present when that function is called on line 2274. Since it's not deducable, that makes line 2274 ill-formed.

The change in the issue is clearly to remove it from the function's template parameters, even if I fudged the order of those parameters.